services/github/schemas/v1/github-rerun-message.yml

```diff
@@ -0,0 +1,59 @@
+$schema: "/schemas/common/metaschema.json#"
+title:        "GitHub Re-run Request Message"
+description: |
+  Message reporting that a GitHub re-run task request has occurred
+type:         object
+properties:
+  version:    {$const: message-version}
+  organization:
+    description: |
+      The GitHub `organization` which had an event.
+    type:         string
+    minLength:    {$const: github-identifier-min-length}
+    maxLength:    {$const: github-identifier-max-length}
+    pattern:      {$const: github-identifier-pattern}
+  repository:
+    description: |
+      The GitHub `repository` which had an event.
+    type:         string
+    minLength:    {$const: github-identifier-min-length}
+    maxLength:    {$const: github-identifier-max-length}
+    pattern:      {$const: github-identifier-pattern}
+  installationId:
+    description: |
+       The installation which had an event.
+    type:          integer
+  eventId:
+    type: string
+    description: |
+      The GitHub webhook deliveryId. Extracted from the header 'X-GitHub-Delivery'
+    pattern: {$const: github-guid-pattern}
+  details:
+    type:         object
+    description: |
+      Metadata describing the push (for version 0)
+  body:
+    type: object
+    description: |
+      The raw body of github event (for version 1)
```

Comment:
Are there properties in the `body` that we should be defining in this schema?

Reply:
I don't think it is needed. At least all other github schemas there don't define it.